### PR TITLE
Add option to have gelu activations

### PR DESCRIPTION
**Pull request by brianhie**
Current version only has silu. Enables config option that uses gelu activations

@exnx

### CODE REVIEW

File: src/layers.py [hunk 1]
```diff
@@ -41,7 +41,14 @@ def __init__(
         super().__init__()
 
         multiple_of = config.get("inner_size_multiple_of", 64)
-        self.act = F.silu
+        self.act_type = config.get("mlp_activation", "gelu")
```
Comment (brianhie) [inline src/layers.py:44]: Reviewing this now, should we actually default to "silu" given that silu is the current default activation? @Zymrael @exnx

Comment (exnx) [reply src/layers.py:44]: wait a min, it's supposed to be gelu, at least that's what the ckpts all use.  This is a source of error accumulation during inference I remember that.  I thought I changed it?

Comment (brianhie) [reply src/layers.py:44]: @exnx it's changed in other repos. not in `togethercomputer/stripedhyena` though.

Comment (exnx) [reply src/layers.py:44]: ah right

Comment (Zymrael) [reply src/layers.py:44]: let's default to silu

Comment (brianhie) [reply src/layers.py]: Done

**Review by Zymrael — APPROVED** [review]

**Pull request merged**